frontend/src/shared/helpers.tsx

```diff
@@ -517,7 +517,13 @@ export const CohortsListColumns: ColDef[] = [
     sortable: false,
   },
   {
-    headerName: "Cohort Complete Date",
+    headerName: "Delivery Date",
+    valueGetter: ({ data }) =>
+      data["hasCohortCompleteCohortCompletes"]?.slice(-1)[0]?.date,
```

Comment:
Because the `CohortComplete` data has already been sorted by `date`, we just need to grab the last `CohortComplete` event (via `.slice(-1)`) to get the oldest one.